### ISSUE TITLE
axe (accessibility audit) - Missing alt text

### ISSUE THREAD

**Issue by katarinazaprazna** [bratislava/olo.sk#824]
<img width="1310" alt="Screenshot 2024-11-02 at 11 29 44" src="https://github.com/user-attachments/assets/f5026566-15f0-4d36-800d-9c2748fe298b">

**Comment by katarinazaprazna** [comment]
Re-check which images are flagged as missing the `alt` attribute
